### PR TITLE
Sprint2 static content

### PR DESCRIPTION
**Pull request by stevezac-osu**
Where do I begin with the changes here:

- Upgraded Strapi from 4.5.6 to 4.6.1
- Upgraded Bootstrap to 5.2.3
- Added Express
  - When you export the content to static content (typically into the ./out folder), something needs to serve it up so it is accessible.
  - Express does this well, so why not make it easy.
  - A new document 'serveout.js' has been added, and you can access the exported content by running `npm run export-serve`
- Added new schema to the backend to accommodate all my changes
- Revamped the _app.js because it was needed in the dynamic process.
- Removed the getLayout functions in each of the /page/ files in favor of a change to the _app.js
- Cleaned up the html on each of the /page/ files
  - One common thing I saw was a `<div className='container-fluid px-0'>` constantly repeated, so I removed it and placed it in the layout
  - Other various small things were also found, but not enumerated here
- **DYNAMIC CONTENT**
  - Nav Bar
  - EVERY PAGE is now dynamic (has the potential to add custom content from the backend)
    - One thing that was challenging was creating dynamic routing appropriately; accordingly, if you want to create a NEW PAGE, you must also create a page in the backend system for the url to be recognized.
  - swapped next.config.js to next.config.mjs and added configurations so dynamic routing would work
  - Accordingly, index.js and about.js have been removed for being superflous.
- **__base model**
  - The base model is now capable of being imported as a standalone ES module (don't ask)
  - Added 3 new filters to the base model:
    - neq: not equal to --- getProjects({id__neq:5}) //would yield all projects that don't have ID equal to 5
    - ew: ends with --- getProject({title__ew:'sound'}) //would yield all projects that have a title that ends in 'sound'
    - in: think (key in array) --- getProject({id___in:[1,2,3]}) //would yield projects with ID 1, 2, and 3.

### CODE REVIEW

**Comment by stevezac-osu** [comment]
Some things I want to note -- Now, when you look at the pages, some of the styles are a little off.  We need to fix these, but getting this code into the main branch will demonstrate that a lot has been done.

File: frontend/pages/_app.js [hunk 1]
```diff
@@ -1,8 +1,26 @@
...
+function MyApp ({ Component, pageProps, router }) {
+  let path = router.query.path
+  if (path === undefined && router.state !== undefined) path = router.state.route
+  const pageCustomContent = <Component {...pageProps} />
+  const page = PageService.getPage({ URLPath__eq: path })
+  let out = ''
+  if (path === '/contactform') {
+    out = <FormLayout><Page page={page} pageCustomContent={pageCustomContent} /></FormLayout>
+  } else {
+    out = <Layout><Page page={page} pageCustomContent={pageCustomContent} /></Layout>
```
Comment (annmariehi) [inline frontend/pages/_app.js:17]: This is so much smarter than doing it page by page.

**Review by annmariehi — COMMENTED** [review]
Everything looks good to me, readjusting the layouts once this is integrated shouldn't be too difficult. Once you take a look at this @nawaji I think its fine to merge.

**Review by nawaji — APPROVED** [review]
Everything looks great! I noticed that some front-end elements look slightly different, but that's probably because of Bootstrap / the dynamic content change (so not a big deal).

**Pull request merged**